website/search/elastic_search.py

```diff
@@ -541,10 +542,10 @@ def update_file(file_, index=None, delete=False):
         'name': file_.name,
         'category': 'file',
         'node_url': node_url,
-        'node_title': file_.node.title,
-        'parent_id': file_.node.parent_node._id if file_.node.parent_node else None,
-        'is_registration': file_.node.is_registration,
-        'is_retracted': file_.node.is_retracted,
+        'node_title': getattr(file_.target, 'title', None),
```

Comment:
Along the lines of my previous comment: better to avoid redundant queries by caching `target` to a variable.

```python
target = file_.target
```